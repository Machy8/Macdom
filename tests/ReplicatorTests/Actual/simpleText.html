@ Replicated text number |
one <br>
two <br>
three <br>
/@

div
	@ Replicated text number |
	one <br>

	two <br>

	three <br>

div
	@ Rewritten replication text number |
	one <br>

	two <br>

	three <br>
