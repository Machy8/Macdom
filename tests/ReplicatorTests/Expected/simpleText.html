Replicated text number one 
<br>
Replicated text number two 
<br>
Replicated text number three 
<br>
<div>
	Replicated text number one 
	<br>
	Replicated text number two 
	<br>
	Replicated text number three 
	<br>
</div>
<div>
	Rewritten replication text number one 
	<br>
	Rewritten replication text number two 
	<br>
	Rewritten replication text number three 
	<br>
</div>
